linode/manager
# upcoming: [DI-20928] - Introduce label for all global filters

## Description 📝
Added label for all ACLP Global filters using filter config

## Changes  🔄

1. Modify name property in FilterConfig with appropriate values
2. Consume name property and use it as a label field all the Autocomplete filters in global filters section
3. Enhanced the common region component to honor the noMarginTop property

## Target release date 🗓️
17-10-2024

## Preview 📷
| Before  | After   |
| ------- | ------- |
|![Screenshot 2024-10-17 at 12 16 04 PM](https://github.com/user-attachments/assets/2e553c2b-2b5f-4747-b2f8-791bb6b9c249)|![Screenshot 2024-10-17 at 12 15 32 PM](https://github.com/user-attachments/assets/a1dac9f6-6c1f-469d-bf0f-56e0e622e1da)|
|![Screenshot 2024-10-17 at 12 16 09 PM](https://github.com/user-attachments/assets/84912abf-ed88-40e7-8551-b99dd20e513f)|![Screenshot 2024-10-17 at 12 15 18 PM](https://github.com/user-attachments/assets/a8f04416-aa3e-401f-ac8d-dde53e1ee7d4)|

## How to test 🧪

1. Login as mock, as still some of the endpoints are not enabled in production.
2. Navigate to monitor tab
4. Select any dashboard
5. You will be able to see label / title for all the filters including dashboard selection and time duration.

## As an Author I have considered 🤔

*Check all that apply*
